README.md

```diff
@@ -5,7 +5,7 @@
 <!-- markdownlint-disable-line MD033 MD034 --><a href="https://raw.githubusercontent.com/redhat-cop/aap_configuration_extended/devel/docs/aap_config_as_code_public_meeting.ics"><img border="0" alt="Google Calendar invite" src="https://www.google.com/calendar/images/ext/gc_button1_en-GB.gif"></a>
 <!-- Further CI badges go here as above -->
 
-This Ansible collection allows for easy interaction with an AWX or Ansible Controller server via Ansible roles using the AWX/Controller collection modules.
+This Ansible collection extends the `infra.aap_configuration` collection by providing extra functionalities that allows advanced operations on the AWX or Ansible Automation Platform Configuration as Code.
```

Comment:
lets remove AWX from this line because it is unclear once the re-arch if AWX will still work with this version or not so we can add it back later if need be

Reply:
done in [new commit](https://github.com/redhat-cop/aap_configuration_extended/pull/25/commits/b8c4cdb0d26e5ea76648aa840f781aaa13f46813)